concepts-and-explanations/visitors.md

```diff
@@ -164,6 +164,8 @@ There are some cases, though, where you may want to share other types of informa
 
 In those cases, you can specify a different, mutable type for your visitor such as in this visitor which counts the number of method invocations in a Java LST:
 
+TODO: This does not work for me. I get errors about `static` declarations not being allowed in inner classes and also that the visitor can't be referenced from a static context.
```

Comment:
```suggestion
TODO(mike-solomon): This does not work for me. I get errors about `static` declarations not being allowed in inner classes and also that the visitor can't be referenced from a static context.
```

Reply:
This isn't going to be committed into the final doc. Was adding this to get feedback from reviewers. Thanks for the suggestion, though!